yjit/src/codegen.rs

```diff
@@ -619,38 +620,36 @@ fn gen_leave_exit(ocb: &mut OutlinedCb) -> CodePtr {
 }
 
 // Generate a runtime guard that ensures the PC is at the expected
-// instruction index in the iseq, otherwise takes a side-exit.
+// instruction index in the iseq, otherwise takes an entry stub
+// that generates another check and entry.
 // This is to handle the situation of optional parameters.
 // When a function with optional parameters is called, the entry
 // PC for the method isn't necessarily 0.
-fn gen_pc_guard(asm: &mut Assembler, iseq: IseqPtr, insn_idx: u16) {
+pub fn chain_entry_guard(
```

Comment:
```suggestion
pub fn gen_entry_chain_guard(
```

To follow the convention and indicate that this method generates code.

Reply:
updated in https://github.com/ruby/ruby/pull/7535/commits/57b567a6e721cef49f494968154f3e9f2a96c653